src/presence_analyzer/utils.py

```diff
@@ -35,6 +38,33 @@ def inner(*args, **kwargs):
     return inner
 
 
+def cache(time_to_live):
+    """
+    Decorator that caches loaded data,
+    expire time is set as datetime.now() + time_to_live.
+    """
+    def cache_decorator(wrapped):
+        """
+        Create cache for wrapped function.
+        """
+        cached = {}
+
+        @wraps(wrapped)
+        def caching(*args):
+            with lock:
+                if args in cached \
+                        and cached[args][1] > datetime.now():
+                    return cached[args][0]
+                else:
+                    result = wrapped()
```

Comment:
where are the arguments gone 😱 it won't work for any func with args, on `get_data` it works since that has no args.